src/site/markdown/maven-artifacts.md.vm

```diff
@@ -149,14 +149,29 @@ file. When you specify the version identifier in this section, you don't have to
 </dependencyManagement>
 ```
 
-To use this with Gradle, an additional [Gradle plugin](https://github.com/spring-gradle-plugins/dependency-management-plugin)
+Gradle 5.0+ supports importing Maven BOM as a [platform](https://docs.gradle.org/current/userguide/platforms.html#sub:bom_import)
+to align dependency versions.
+
+`build.gradle`
+
+```
+dependencies {
+  implementation platform('org.apache.logging.log4j:log4j-bom:${Log4jReleaseVersion}')
+
+  implementation 'org.apache.logging.log4j:log4j-api'
+  implementation 'org.apache.logging.log4j:log4j-core'
```

Comment:
I would use `runtime` as scope for `log4j-core`.

Reply:
Do you think this documentation should describe the common case, or advanced component development configurations?

Reply:
I'm more inclined to describe common case. If it's ok with you, I'll change it to runtimeOnly.

Reply:
@grossws, the documentation you cited might be misleading: if someone develops custom plugins, `log4j-core` needs to be a compile dependency because custom plugins must reference implementation classes. As it is written right now someone might think that adding Log4j2 Core as compile dependency improves startup.

I would remove that remark from the docs.

Reply:
Updated with note for Gradle 2.8-3.3 about compile/runtime instead of implementation/runtimeOnly